modules/openapi-generator/src/main/resources/php-symfony/composer.mustache

```diff
@@ -25,7 +25,7 @@
         "ext-mbstring": "*",
         "symfony/validator": "*",
         "jms/serializer-bundle": "*",
-        "symfony/framework-bundle": "^2.3|^3.0|^4.1"
```

Comment:
Instead of removing 2.3 I would bump it to 2.7. It should work as well, and there is a ton of 2.x projects out there still. Plus 2.7 is still receiving security updates. No need to break compat as long as it costs nothing.